java-algo/src/main/java/com/c1games/terminal/algo/FrameData.java

```diff
@@ -72,26 +72,26 @@ public PlayerUnitList(List<PlayerUnit> contents) {
         }
     }
     public static final class PlayerUnits {
-        public PlayerUnitList filter;
-        public PlayerUnitList encryptor;
-        public PlayerUnitList destructor;
-        public PlayerUnitList ping;
-        public PlayerUnitList emp;
-        public PlayerUnitList scrambler;
+        public PlayerUnitList wall;
+        public PlayerUnitList support;
+        public PlayerUnitList turret;
+        public PlayerUnitList scout;
+        public PlayerUnitList demolisher;
+        public PlayerUnitList interceptor;
         public PlayerUnitList remove;
         public PlayerUnitList upgrade;
 
         @Override
         public String toString() {
             return "PlayerUnits{" +
-                    "filter=" + filter +
-                    ", encryptor=" + encryptor +
-                    ", destructor=" + destructor +
-                    ", ping=" + ping +
-                    ", emp=" + emp +
-                    ", scrambler=" + scrambler +
+                    "wall=" + wall +
+                    ", support=" + support +
+                    ", turret=" + turret +
+                    ", scout=" + scout +
+                    ", demolisher=" + demolisher +
+                    ", interceptor=" + interceptor +
                     ", remove=" + remove +
-                    ", remove=" + upgrade +
+                    ", upgrade=" + upgrade +
```

Comment:
This is a minor error correction: This incorrectly would display remove= instead of upgrade